features/support/world.js

```diff
@@ -100,6 +100,7 @@ function CustomWorld({ attach, parameters }) {
     this.instance = `${this.protocol}://${process.env.SD_API}`;
     this.testOrg = process.env.TEST_ORG;
     this.username = process.env.TEST_USERNAME;
+    this.scmContext = process.env.TEST_SCM_CONTEXT;
```

Comment:
Need to add it to the readme too. https://github.com/screwdriver-cd/screwdriver/blob/master/README.md#functional-tests